src/frontend/Komponenter/Oppgavebenk/OppgaveRad.tsx

```diff
@@ -24,10 +24,11 @@ const Flatknapp = hiddenIf(Knapp);
 const kanJournalføres = (oppgave: IOppgave) => {
     const { behandlesAvApplikasjon, behandlingstema, oppgavetype } = oppgave;
     return (
-        behandlesAvApplikasjon === 'familie-ef-sak-førstegangsbehandling' &&
+        (behandlesAvApplikasjon === 'familie-ef-sak-førstegangsbehandling' ||
+            behandlesAvApplikasjon === 'familie-ef-sak') &&
         oppgavetype === 'JFR' &&
         behandlingstema &&
-        ['ab0071', 'ab0177', 'ab0028'].includes(behandlingstema)
```

Comment:
Er `'ab0177', 'ab0028'` de gamle enhetene?

Reply:
Nei, skolepenger og barnetilsyn - tenker at vi fjerner dette fra journalføring nå som vi skal i prod og disse to ikke er aktuelle før i 2022...